src/pages/Beta/Pool/index.tsx

```diff
@@ -148,9 +154,11 @@ const PoolUI = () => {
             <Wallet activeMenu={activeMenu} setMenu={handleSetMenu} menuItems={menuItems} />
           )}
         </Box>
-
         <Hidden upToSmall={true}>
           <Box>
+            <ExternalLink onClick={() => setAddLiquidityModalOpen(true)} style={{ cursor: 'pointer' }}>
```

Comment:
will this display in desktop and mobile both ?

Reply:
No, we need a design for mobile